cpp/src/arrow/engine/substrait/ext_test.cc

```diff
@@ -198,11 +207,11 @@ TEST(ExtensionIdRegistryTest, RegisterTempFunctions) {
 
     for (util::string_view name : kTempFunctionNames) {
```

Comment:
I know it is not part of your edit, should this be `const util::string_view& name` instead?

Reply:
I linked an article earlier that talks about why it is better to pass `util::string_view` by value.  I find it is helpful to think of `string_view` as just a specialized `char*`.  So I wouldn't do `const char*& name` and I don't do `const util::string_view& name`

Reply:
:+1: